var/spack/repos/builtin/packages/fastmath/package.py

```diff
@@ -0,0 +1,58 @@
+##############################################################################
```

Comment:
Well, I kinda figure its no worse than having a small patchfile there. I agree it _could_ be kinda confusing to other Spack develpers or anyone who goes wondering into fastmath package dir and sees that .tar.gz file.

FWIW, `spack install fastmath` is IMHO _much_ preferred to `spack install --only=dependencies fastmath`. So, I don't really like that solution. Any  way to have a package advertise --only=dependencies as its default behavior?